### PR TITLE
Make sure config exists when generating dashboard

### LINKED ISSUE #12372: null is not an object (evaluating 'e.hass.config.location_name')
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

error in the Inspector console

### Describe the behavior you expected

no error

### Steps to reproduce the issue

1. really dont know, happening on all pages
2.
3.
...

### What version of Home Assistant Core has the issue?

2022.5.0.dev20220421

### What was the last working version of Home Assistant Core?

?

### In which browser are you experiencing the issue with?

Safari, Chrome

### Which operating system are you using to run this browser?

MacOS  Monterey 12.2.1

### State of relevant entities

_No response_

### Problem-relevant frontend configuration

_No response_

### Javascript errors shown in your browser console/inspector

```txt
TypeError: null is not an object (evaluating 'e.hass.config.location_name')
(anonieme functie) — 14f311db.js:1:12122
asyncFunctionResume
(anonieme functie) — ha-app-layout.js:101
asyncFunctionResume
(anonieme functie)
promiseReactionJobWithoutPromise
promiseReactionJob
```

### Additional information

Safari:
Schermafbeelding 2022-04-21 om 11 54 17

Chrome:
Schermafbeelding 2022-04-21 om 11 59 24

might be related to Browser_mod, though not sure of the cause/messenger

see: https://github.com/thomasloven/hass-browser_mod/issues/318

### PR DESCRIPTION
**Pull request by zsarnett**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #12372
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/panels/lovelace/strategies/original-states-strategy.ts [hunk 1]
```diff
@@ -86,7 +86,7 @@ export class OriginalStatesStrategy {
     info: Parameters<LovelaceDashboardStrategy["generateDashboard"]>[0]
   ): ReturnType<LovelaceDashboardStrategy["generateDashboard"]> {
     return {
-      title: info.hass.config.location_name,
```
Comment (balloob) [inline src/panels/lovelace/strategies/original-states-strategy.ts:89]: When is this not the case?

**Review by balloob — APPROVED** [review]

File: src/panels/lovelace/strategies/original-states-strategy.ts (hunk 1 above)
Comment (zsarnett) [reply src/panels/lovelace/strategies/original-states-strategy.ts:89]: Not sure exactly. Possibly when not in storage mode?

Comment (balloob) [reply src/panels/lovelace/strategies/original-states-strategy.ts:89]: No. `config` is part of the `hass` object and we do not show the UI unless it's loaded: https://github.com/home-assistant/frontend/blob/dev/src/layouts/home-assistant.ts#L69

Comment (zsarnett) [reply src/panels/lovelace/strategies/original-states-strategy.ts:89]: Not sure then how this error is even showing. I will leave this open and try to trace it back to find the root issue

**Comment by Mariusthvdb** [comment]
is there any reason this isnt merged yet? just letting you know the issue is still very much alive in  2022.6.0.dev20220522/frontend20220521.0 

please let me know if we can do anything to test run this or other action is required for you.

**Comment by bramkragten** [comment]
If you can reproduce this without any custom cards, please let us know how :-)

**Comment by zsarnett** [comment]
Yes. Sorry meant to get back to this. The fix in the PR is just knowing there is an error that config doesn't get to this point. The issue is that config should ALWAYS be there at this point so it doesn't make sense. We should fix the underlying issue in stead of band-aiding this

**Comment by Mariusthvdb** [comment]
without any resource loaded (and only card_mod and browser_mod via the frontend method, I can still see the issue happening. It seems to only happen when navigating to a fresh view, which isnt in cache yet. Though I admit not being 100% certain. 
Can you give me a pointer what the `hass.config.location_name` in fact is? I can not enter it in console so a bit hard to debug this

Schermafbeelding 2022-05-25 om 11 10 14

Ill reinstall the resources (they apparently didnt cause it) and remove card_mod/browser_mod once more. (did that before and the console still showed it)

**Pull request closed without merging**